bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v1/CarouselImplTest.java

```diff
@@ -64,11 +68,11 @@ void testEmptyCarousel() {
     void testCarouselWithItems() {
         Carousel carousel = getCarouselUnderTest(CAROUSEL_1);
         Object[][] expectedItems = {
-                { "item_1", "Teaser 1", "core/wcm/components/teaser/v1/teaser",
+                { "item_1", "Teaser 1", "cq:Component/item",
```

Comment:
shouldn't it be core/wcm/components/teaser/v1/teaser/cta?

Reply:
@jckautzmann this is testing the carousel, there it's /item, no ? Teaser just happend to be the old entry for  @type in that carousel item.